pkg/rng/entropy.go

```diff
@@ -109,7 +108,7 @@ func setAvailableTRNG() error {
 // UpdateLinuxRandomness seeds random data from
 // /dev/hwrng into /dev/random based on a timer and
 // the entropy pool size
-func UpdateLinuxRandomness(recoverer recovery.Recoverer) error {
+func UpdateLinuxRandomness() error {
```

Comment:
Did 9elements originally write these packages? I'm a bit hesitant to accept these changes because only systemboot uses this package and pkg/recovery. We (goog) accepted these under the premise that we'd eventually somewhat unify, but I'm hesitant to just go and rewrite this stuff without some sort of unification and without talking to them.

Reply:
I restored the public API on this function again